test/helpers/convertDataToBytes.js

```diff
@@ -1,11 +1,12 @@
 const convertDataToBytes = (data) => {
+  const numColons = 2
   let concatenatedString = ''
 
   Object.values(data).forEach((value) => {
     concatenatedString += `${value}::`
   })
 
-  const buffer = Buffer.from(concatenatedString.substring(0, concatenatedString.length - 3))
+  const buffer = Buffer.from(concatenatedString.substring(0, concatenatedString.length - numColons))
```

Comment:
Hah! This wouldn't have been a problem if you had done:

```javascript
const concatenatedString = Object.values(data).join('::')
```

;)

Reply:
(Also, this is a little more brittle now because `numColons` isn't actually dictating how many colons you add on line 6.)

Reply:
Haha dammit there's all these little utility functions that exist but I never use and forget about. I like `join`!